pandas/tseries/base.py

```diff
@@ -17,6 +17,26 @@
 import pandas.algos as _algos
 
 
+
+class DatelikeOps(object):
+    """ common ops for DatetimeIndex/PeriodIndex, but not TimedeltaIndex """
+
+    def strftime(self, date_format):
+        """
+        Return a list of formmatted strings specified by date_format
+
+        Parameters
+        ----------
+        date_format : str
+            date format string (e.g. "%Y-%m-%d")
+
+        Returns
+        -------
+        a list formatted strings
+        """
+        return self.format(date_format=date_format)
```

Comment:
Yes, but for a user, that is an implementation detail. Having a DatetimeIndex and calling `idx.strftime(format)` returning a list is strange I think. But I am not sure what it should be, at least an array, but maybe just a Index?